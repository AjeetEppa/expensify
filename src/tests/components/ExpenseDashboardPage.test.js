import { expect } from "@jest/globals";
import { shallow } from "enzyme";
import React from "react";
import ExpenseDashboardPage from "../../components/ExpenseDashBoardPage";

test("should render ExpenseDashBoardPage correctly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
