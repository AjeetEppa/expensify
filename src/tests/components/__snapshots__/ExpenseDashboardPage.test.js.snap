// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashBoardPage correctly 1`] = `
<div>
   
  <p>
    This is dashboard component
  </p>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
